### DISCUSSION TITLE
How to add a label to all app containers by default?

### DISCUSSION THREAD

**Discussion by rhyst** [dokku/dokku#5452] [Q&A]
Hello, is there a way to add a label to all app container by default? Or a way to specify a label for an app during deployment?

Specifically I want to prevent watchtower from checking my dokku apps because they have no valid upstream so I need to add the label:

```
com.centurylinklabs.watchtower.enable=false
```

**Comment by rhyst** [comment]
Ah I have answered half of my question. I can use the docker-options command to set labels on deployment:

```
dokku docker-options:add app-name deploy  "--label=com.centurylinklabs.watchtower.enable=false"
```

But I still would like to know if there's a way to set a default option for all new apps.

**Comment by josegonzalez** [comment]
There is no way to set global properties or docker-options at this time. You can create your own plugin that implements the trigger [`docker-args-process-deploy`](https://dokku.com/docs/development/plugin-triggers/#docker-args-process-deploy) to inject that label everywhere.

Documentation on plugin creation [here](https://dokku.com/docs/development/plugin-creation/).
